servers/visual/shader_language.cpp

```diff
@@ -1931,7 +1931,8 @@ bool ShaderLanguage::_validate_function_call(BlockNode *p_block, OperatorNode *p
 		}
 
 		if (!fail) {
-			p_func->return_cache = pfunc->return_type;
+			if (r_ret_type)
```

Comment:
That shouldn't be here, it seems you merged your own commit with 834112772b9cb1f7c1ca04b43ab26e23acb149bc (which explains why @reduz is shown as commit author instead of you).